crates/bin/signer/src/main.rs

```diff
@@ -38,6 +38,9 @@ impl signer::Signer for SignerState {
         declare_keyset_request: Request<DeclareKeysetRequest>,
     ) -> Result<Response<DeclareKeysetResponse>, Status> {
         let declare_keyset_request = declare_keyset_request.get_ref();
+        if declare_keyset_request.max_order >= 64 {
```

Comment:
https://github.com/nutty-raccoon/paynet/issues/50#issue-2902273418

I say it takes at most 64 values.
So it should be > not >=